src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,61 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+type Props = {
+  onFilterByQuery: (event: string) => void;
+  onFilterBy: (event: string) => void;
+  query: string;
+};
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+export const TodoFilter: React.FC<Props> = ({
+  onFilterByQuery = () => {},
+  onFilterBy = () => {},
```

Comment:
```suggestion
  onFilterByQuery,
  onFilterBy,
```
These props are required, so it doesn't make sense to assign default values

Reply:
> These props are required, so it doesn't make sense to assign default values

Not fixed, and a lot of other comments too 🤷